src/App.js

```diff
@@ -1,16 +1,48 @@
 import React from 'react';
+import TodoForm from './components/TodoComponents/TodoForm';
+import TodoList from './components/TodoComponents/TodoList';
 
 class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor(){
+    super();
+    this.state={
+      textInput: '',
+      todos: []
+    };
+  }
+
+  handleInput = (event) => {
+    this.setState({
+      textInput: event.target.value
+    });
+  };
+
+  handleClick = (event) => {
+    event.push();
```

Comment:
So what you may want to do is make an array that is a copy of the todolist, push whatever is in the form to that copy, then setstate to overwrite the todolist with the copy.